perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -411,7 +416,7 @@ def Prepare(benchmark_spec):
   logging.info('Umount scratch disk on %s at %s', vm, mount_point)
   vm.RemoteCommand('sudo umount %s' % mount_point)
 
-  if FLAGS.device_fill_size is not '0':
+  if FLAGS.prefill_device:
```

Comment:
I made a PR to fix this: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/516